llvm/include/llvm/Support/PropertySetIO.h

```diff
@@ -185,6 +185,8 @@ class PropertySetRegistry {
       "SYCL/specialization constants";
   static constexpr char SYCL_COMPOSITE_SPECIALIZATION_CONSTANTS[] =
       "SYCL/composite specialization constants";
+  static constexpr char SYCL_SPEC_CONSTANTS_DEFAULT_VALUES[] =
```

Comment:
Given that older spec constants are experimental feature and the 2020 are not there yet, I'd suggest to (logically) group spec constants under one category for more ordered structure in future: "SYCL/specialization constants", with three sub-categories - SYCL/specialization constants/names, SYCL/specialization constants/composite names, SYCL/specialization constants/default values. (Category names here are sample)

Reply:
Actually, SYCL_COMPOSITE_SPECIALIZATION_CONSTANTS is never used in the code. Given that we have only 2 sub-categories, I think we don't need to group them somehow else.

Reply:
2 sub-categories still form a group :) But I don't feel strong against having them as separate categories, so should be OK as is too.

Also, please remove SYCL_COMPOSITE_SPECIALIZATION_CONSTANTS then